ctsit/m3c-web
# Add "Recent Publications" and "Recent Projects" list widgets in place of the images at the bottom of the index page

This change affects the index.html page of the site. Previously, the space below the slider was occupied by visuals and an About box. I have replaced those visuals with a widget containing two lists of the 10 most recent publications (sorted by date) and 10 most "recent" projects (sorted by id number).

This change keeps the front page up-to-date so that users can quickly see what's currently happening in the community.

## Verification

1. Open the index.html file and attach the following end point to the url: 
`?endpoint=http%3A%2F%2Fprod.vivo.metabolomics.info%2Ftpf%2Fcore`
2. Scroll to the bottom of the page, below the "Spotlight" slider.
3. Verify that there are 10 publications and 10 projects listed, and that they are arranged by the most recent date and highest id number.

- [x] The lists link back to the publications/projects search pages, while individual items in each list link to their specific page within the people portal. The widget loads properly and scales appropriately for screen size (the lists are stacked on top of each other for mobile device screens, just like the slider items).
- [x] The lists do not affect anything else on the web site.

### Screenshots

After
![after](https://user-images.githubusercontent.com/54954488/72277450-f0234f80-35ff-11ea-8652-816b1c5adcf2.png)


Before
<img width="1440" alt="before" src="https://user-images.githubusercontent.com/54954488/72083726-4b87d180-32d0-11ea-852c-204b03b8d70b.png">



### Browsers Checked

- [x] Safari 12.1.2 on macOS Mojave
- [x] Chrome 77.0.3865.120 on macOS Mojave
- [x] Firefox 69.0.3 on macOS Mojave
- [x] Internet Explorer 11 on Windows 10


## Affirmations

All of these should have a check by them. Any exception requires an explanation.

* [x] I used JSDocs for documentation and JSDocs-based type hinting.
* [x] I added and updated any relevant documentation such as the `README` or `CHANGELOG`?
* [x] I have made an effort to minimize code changes and have not included any cruft (old comments, print-debugging, unused variables).
